lib/web_console/view.rb

```diff
@@ -25,9 +25,13 @@ def render_inlined_string(template)
       render(template: template, layout: 'layouts/inlined_string')
     end
 
-    # Escaped alias for "ActionView::Helpers::TranslationHelper.t".
+    # Override method for "ActionView::Helpers::TranslationHelper.t".
```

Comment:
The notation for instance methods is #t: `ActionView::Helpers::TranslationHelper#t`.